Formula/elasticsearch@5.0.2.rb

```diff
@@ -0,0 +1,130 @@
+class ElasticsearchAT502 < Formula
+  desc "Distributed search & analytics engine"
+  homepage "https://www.elastic.co/products/elasticsearch"
+  url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.0.2.tar.gz"
+  sha256 "bbe761788570d344801cb91a8ba700465deb10601751007da791743e9308cb83"
+
+  head do
+    url "https://github.com/elasticsearch/elasticsearch.git"
+    depends_on "gradle" => :build
+  end
+
```

Comment:
The `head` block is not needed, one would use the main `elasticsearch` formula to build from head.